### DISCUSSION TITLE
Best way to add custom pages to admin ui

### DISCUSSION THREAD

**Discussion by shahidcodes** [pocketbase/pocketbase#3905] [Q&A]
I have started using pocketbase and loving it. However, I am presented with a problem that can not be solved with current pocketbase admin UI. I need to show some charts and stats in the admin ui. I saw that pocketbase uses svelte and I think I can do that but I am afraid I might break updates from main. 

Can you please advise how should I do that I can continue to receive updates?

**Comment by ganigeorgiev** [comment]
The Admin UI is not customizable at the moment and cannot be easily extended or changed without forking the project.

This may change eventually in the future with https://github.com/pocketbase/pocketbase/issues/898, but it will be done after first refactoring and stabilizing the Go internals (_which most likely will happen after the release of my other open source project that I'm currently working on_).
